test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelCard.tsx

```diff
@@ -38,19 +36,19 @@ export class IModelCard extends React.Component<IModelCardProps, IModelCardState
   // called when this component is first loaded
   public override async componentDidMount() {
     // we don't get the thumbnail until it's needed.
-    if (!this.props.iModel.thumbnail)
-      this.startRetrieveThumbnail(this.props.iModel); // eslint-disable-line @typescript-eslint/no-floating-promises
+    // if (!this.props.iModel.thumbnail)
```

Comment:
I assume this also should either be deleted or uncommented

Reply:
Yes, probably deleted since the client doesn't support thumbnails yet (or at least I far as I saw) and I don't think it's necessary to re-write it against the rest APIs directly.